requirements.txt

```diff
@@ -1,9 +1,8 @@
-requests[security]
+-e .
```

Comment:
PyJWT and cryptography is locked in this file for python versions but not in setup.py but maybe they should?

Reply:
For PyJWT, if there's a lower bound we're aware of for the functionality we're using we could include that. Otherwise I'd rather omit it.

Good idea to include the `cryptography<3.4;python_version<"3.7"` line though, like we do in `dash[testing]`, IIRC it's tough to figure out the problem if that's not specified.